### PR TITLE
Add dashboard export examples

### PR DESCRIPTION
**Pull request by avanbrunt-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

CBAPI-2581

## Pull Request Description

Adding two new examples scripts that replicate legacy dashboard export behavior

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

Manual testing on the example scripts

### CODE REVIEW

File: examples/platform/attacks_detected_stopped_export.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                    stopped.write(f'{sep.join(list(ttps))},')  # TTPs
+                    stopped.write(f'{alert.threat_cause_threat_category},')  # ThreatCategory
+                    stopped.write(f'{alert.run_state},')  # RunState
+                    stopped.write(f'{alert.policy_applied},')  # PolicyAppliedState
+                    stopped.write(f'{False if alert.workflow["state"] else True},')  # Dismissed
+                    stopped.write(f'{alert.threat_cause_reputation},')  # Reputation
+                    stopped.write('\n')
+
+        if len(alerts_detected) > 0:
+            print(f'Writing Attacks Stopped to: attacks_detected_{now}.csv')
```
Comment (abowersox-cb) [inline examples/platform/attacks_detected_stopped_export.py:109]: Should be "Writing Attacks Detected..." for consistency.

**Review by abowersox-cb — COMMENTED** [review]
Overall pretty straightforward. One wording change noted for consistency.

File: examples/platform/attacks_detected_stopped_export.py (hunk 1 above)
Comment (avanbrunt-cb) [reply examples/platform/attacks_detected_stopped_export.py]: Thanks good catch

**Review by abowersox-cb — APPROVED** [review]

**Review by emitreva-cb — APPROVED** [review]
LGTM

**Comment by hkaragitliev-cb** [comment]
Just wondering if we should "enforce" that the scripts are being executed with python3 -  with a shebang #!/usr/bin/env python3 ?

**Review by hkaragitliev-cb — APPROVED** [review]

**Pull request merged**